### ISSUE TITLE
Can't create a script for battle.

### ISSUE THREAD

**Issue by Seph501** [Fate-Grand-Automata/FGA#1043]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1589

### Describe the bug

When I try to create a script for battle at Battle Configs and then typing the "+" button, the app freezes and after like 5 seconds appears a message saying "the app isn't working" and have to close it, so, can't create a script (as I said) and start auto battle.

### Video

https://youtu.be/aRt0d7jws9Y

### Device model

Huawei Y9s

### Android version

10

### RAM

4Gb

**Comment by reconman** [comment]
Reinstall doesn't help, right? Otherwise I would need you to gather the crash log using `adb logcat -E > log.txt` and then copy the part with "mathewsachin" in it.

Should look like my stacktrace in #1049, but more unreadable.

**Comment by Seph501** [comment]
Still doesn't work, no matter what I change and dunno how to do the adb logcat -E and that stuff on cellphone :(

**Comment by reconman** [comment]
You can't really, you need a computer and google for that.

**Comment by Seph501** [comment]
So that means I can't use the app on my cellphone? :(

**Comment by reconman** [comment]
We can't help you unless we see some logs. You don't have a PC?

**Comment by Seph501** [comment]
Yeah, have one, so, have to download the apk there and do the logcat thing?

**Comment by Seph501** [comment]
I don't understand how to do all that stuff hahaha guess I'll just have to play manually :/

**Comment by reconman** [comment]
See https://forum.xda-developers.com/t/tool-minimal-adb-and-fastboot-2-9-18.2317790/ for a guide on how to install adb.

`adb logcat -E` then is just something you can run in the Windows command line. You can also find some general guides for adb if you search for them.

**Comment by Seph501** [comment]
Ok, thanks, gonna check it.

Just one more question before that, if I do that on PC, how can I play then in my cellphone?

**Comment by reconman** [comment]
adb is just a tool for developers to install things on your phone or inspect logs in this case.

What I want you to do is
1. Connect your phone to your USB via PC
2. Enable Android Debug Bridge (adb) on your phone
3. Make FGA crash
4. Run the `adb logcat -E > log.txt` command in order to transfer the logs to a text file in your current directory
5. Upload the txt file here

**Comment by Seph501** [comment]
Ooooohh got it, thanks.

Ok, gonna try that.

**Comment by Seph501** [comment]
Haven't been able to do that (also, idk how xD) is it really necessary? :/

**Comment by reconman** [comment]
Have you ever tried reinstalling FGA?

**Comment by Seph501** [comment]
Yeah, tried like, 2-3 times a Month :/

**Comment by Seph501** [comment]
Do I have to download or do something with those?

![Screenshot_20220526_185638_com android chrome](https://user-images.githubusercontent.com/96612473/170602644-76954ecf-65be-42a9-8811-eaa210a161da.jpg)

**Comment by reconman** [comment]
Regarding the crash: You're the only one where this happens, so that's why I asked you to use adb.

What also worked in the past is an app called Logcat Reader where you have to seach for "mathew" and then press ... -> save to create a log file.

Regarding what to do with the links, how did you install FGA in the first place without that knowledge? The first link is the latest app version.

**Comment by Seph501** [comment]
Well, I started using FGA with the "past" version (the one with red icon) then, the update came (with the full screen FGO version), that's all haha

Ok, let me try with that app, hope that works :/

**Comment by Seph501** [comment]
Already downloaded it but again, with a PC and stuff that I don't even understand... 
![Screenshot_20220526_192217_com dp logcatapp](https://user-images.githubusercontent.com/96612473/170604602-339168b6-2f43-49d0-b35a-c25ebc48822c.jpg)

**Comment by reconman** [comment]
There are tons of adb guides in text and video form. Search for them on Google or YouTube.

**Comment by Seph501** [comment]
Hello, it's me again, sorry.

Started playing on PC and it runs there, but I have some little problems, is there any way I can contact you faster? Event is about to end and I need lots of QP haha

**Comment by Seph501** [comment]
Please, I need help :(

**Comment by reconman** [comment]
I can't do personal debug sessions with every FGA user. If you post details about your setup and problem (screenshots, videos and descriptions), I can tell you if it's a known issue.

**Comment by Seph501** [comment]
Ok, let me make a video, I guess it's something "simple".

The program don't chose a Supp (I have set Skadi MLB 10/10/10

**Comment by Seph501** [comment]
Ok, let me make a video, I guess it's something "simple".

The program don't chose a Supp (I have set Skadi MLB 10/10/10 with Bella Lisa MLB).

Then, on 2nd wave, it does the skill/np command, but if it doesn't kill the wave, it activates skills from 3rd wave.

And finally, it doesn't restart the quest, ends but don't start and select the supp.

Give some minutes and I'll try making a video
